app/src/main/java/org/mozilla/focus/widget/Cursor.java

```diff
@@ -50,14 +54,23 @@ public void run() {
     };
 
 
+    private Handler cursorHandler = new Handler();
+    private Runnable cursorTick = new Runnable() {
```

Comment:
I think we can use `ViewPropertyAnimator.setStartDelay` instead so we don't need to deal with Handlers/Loopers. Don't forget to cancel the animator in `onStart/onStop`